optaplanner-integration/optaplanner-spring-boot-autoconfigure/pom.xml

```diff
@@ -28,6 +28,11 @@
       <artifactId>optaplanner-core</artifactId>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>org.optaplanner</groupId>
+      <artifactId>optaplanner-test</artifactId>
+      <optional>true</optional>
+    </dependency>
```

Comment:
-1. `optaplanner-test` ends up in `BOOT-INF/lib/` of the executable fat JAR built by `spring-boot-maven-plugin`.

Reply:
I presume that was not the intention and the `optional` scope is there to avoid that, but it doesn't as it seems.

Reply:
Wrong line. Disregard this thread.

Reply:
Even when optional=true?

Reply:
Do note that it's not optional=true for the starter... I presume you mean the entry there that is a -1?